clients/src/main/java/org/apache/kafka/common/requests/DescribeQuorumRequest.java

```diff
@@ -86,10 +86,10 @@ protected Struct toStruct() {
 
     @Override
     public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
-        return getErrorResponse(this.data, Errors.forException(e));
+        return new DescribeQuorumResponse(getPartitionLevelErrorResponse(this.data, Errors.forException(e)));
```

Comment:
Seems like it would be reasonable to use `getTopLevelErrorResponse` here since there is just one error code. (Same question for other APIs)